### PR TITLE
Standard Info NPC for Chocobo Circuit

### PR DESCRIPTION
**Pull request by Tom-Neverwinter**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

adds script for standard info npcs

## Steps to test these changes

talk to npc's

talk to npcs

### CODE REVIEW

File: scripts/zones/Chocobo_Circuit/npcs/Amaduralle.lua [hunk 1]
```diff
@@ -1,7 +1,8 @@
 -----------------------------------
 -- Area: Chocobo_Circuit
---  NPC: Amaduralle
+-- NPC: Amaduralle
 -- Standard Info NPC
+-- !pos -369.286 -4.000 -464.873
```
Comment (zach2good) [inline scripts/zones/Chocobo_Circuit/npcs/Amaduralle.lua]: Can you make sure each of these commands also has the zone number, so you can copy/paste it and go to the NPC from anywhere

File: scripts/zones/Chocobo_Circuit/npcs/Synergy_Engineer.lua [hunk 2]
```diff
@@ -0,0 +1,23 @@
+-----------------------------------
+-- Area: Chocobo Circuit
+-- NPC: Synergy Engineer
+-- Type: Standard NPC
+-- !pos -325.376,0.000,-524.698
+-- !pos -304.635,0.000,-516.724
```
Comment (zach2good) [inline scripts/zones/Chocobo_Circuit/npcs/Synergy_Engineer.lua]: Duplicated, and commas still here

**Review by zach2good — COMMENTED** [review]
Getting quite close 👍

File: scripts/zones/Chocobo_Circuit/npcs/Amaduralle.lua (hunk 1 above)
Comment (Tom-Neverwinter) [reply scripts/zones/Chocobo_Circuit/npcs/Amaduralle.lua]: done :)

File: scripts/zones/Chocobo_Circuit/npcs/Synergy_Engineer.lua (hunk 2 above)
Comment (Tom-Neverwinter) [reply scripts/zones/Chocobo_Circuit/npcs/Synergy_Engineer.lua]: done :)

**Review by zach2good — APPROVED** [review]

**Pull request merged**